src/components/tabs/tabs.component.tsx

```diff
@@ -8,21 +8,60 @@ import React, {
   createRef,
   cloneElement,
   Children,
+  ReactElement,
 } from "react";
-import PropTypes from "prop-types";
-import styledSystemPropTypes from "@styled-system/prop-types";
-import Tab from "./tab";
+import { MarginProps } from "styled-system";
+import Tab, { TabProps } from "./tab";
```

Comment:
It looks like you're importing `TabProps` just to reexport it from this file. I'd rather export it directly from the `/.tab` in the `index.ts` file

Reply:
I'd overlooked that, thanks. I must have been using it at some point, but as it is I agree it's pointless to import just to reexport.

Reply:
Actually now I come to look at this, it is being used - on line 95, to assert that the `filteredChildren` have type `ReactElement<TabProps>`. But I now see that it's only necessary to assert it's a `ReactElement` (otherwise it complains about `child.props`), so I'll make that change and export `TabProps` directly as you suggest.